docs/core/extensions/generic-host.md

```diff
@@ -26,7 +26,7 @@ The main reason for including all of the app's interdependent resources in one o
 
 The host is typically configured, built, and run by code in the `Program` class. The `Main` method:
 
-- Calls a <xref:Microsoft.Extensions.Hosting.Host.CreateDefaultBuilder> method to create and configure a builder object.
+- Calls a <xref:Microsoft.Extensions.Hosting.Host.CreateApplicationBuilder> method to create and configure a builder object.
```

Comment:
In the second instance the xref ends in %2A -- for consistency, we could add it here. The difference is that not adding it renders () after the method name.

```suggestion
- Calls a <xref:Microsoft.Extensions.Hosting.Host.CreateApplicationBuilder%2A> method to create and configure a builder object.
```